cli.sh

```diff
@@ -283,6 +283,15 @@ case $1 in
     checkroot
     usb "$2"
     ;;
+  sdcard)
```

Comment:
Any particular reason why the case statements are in `cli.sh` as opposed to `sd_burner.sh`?

Reply:
If you check `cli.sh`, we've removed all the individual sourcing so I guess this doesn't even need to be here anymore.